src/app/lib/config/services/bengali.js

```diff
@@ -222,11 +222,11 @@ export const service = {
           text: 'প্রিভেসি নীতি',
         },
         {
-          href: 'https://www.bbc.com/bengali/institutional-37289967',
+          href: 'https://www.bbc.com/usingthebbc/cookies/',
```

Comment:
I missed this change initially 🤦 , but where did you get the requirement to update this link?

Reply:
I checked with Jon about this link since it was the same as for "Contact us". There was a mistake on the website previously and Jon confirmed that this should be a cookies link